cartridge-cli.lua

```diff
@@ -2438,6 +2452,8 @@ local function app_pack(args)
     else
         die("Unknown package type: %s", args.type)
```

Comment:
In case of an error, we do not clear the directory... But I suppose that we should clean up any build garbage in case of error